hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/interfaces/BlockManager.java

```diff
@@ -34,7 +34,7 @@ public interface BlockManager {
    * Puts or overwrites a block.
    *
    * @param container - Container for which block need to be added.
-   * @param data     - Block Data.
+   * @param data - Block Data.
    * @return length of the Block.
    * @throws IOException
```

Comment:
Please also remove such `@throws` tags that only mention the exception type, but not when or why it would be thrown.  The same information is already known from the methods' `throws` declaration.

Reply:
Thank you for the review! I have made the changes in the next commit. Please check if it's fine now.